code/game/machinery/computer/arcade/arcade_game.dm

```diff
@@ -0,0 +1,359 @@
+				"enemy_hp" = enemy_hp,
+				"enemy_max_hp" = enemy_max_hp,
+				"enemy_mp" = enemy_mp,
+				"enemy_max_mp" = enemy_max_mp,
+				"gameover" = gameover,
+				"blocked" = blocked,
+				"arcade_type" = type,
+				)
+
+/datum/arcade_game/space_villain/import_data(var/list/args)
+	if(!..())
+		return
+	name = args["name"]
+	emagged = args["emagged"]
+	enemy_name = args["enemy_name"]
+	temp = args["temp"]
+	player_hp = args["player_hp"]
+	player_max_hp = args["player_max_hp"]
+	player_mp = args["player_mp"]
+	player_max_mp = args["player_max_mp"]
+	enemy_hp = args["enemy_hp"]
+	enemy_max_hp = args["enemy_max_hp"]
+	enemy_mp = args["enemy_mp"]
+	enemy_max_mp = args["enemy_max_mp"]
+	gameover = args["gameover"]
+	blocked = args["blocked"]
+
+/datum/arcade_game/space_villain/get_dat()
+	var/dat = "<a href='byond://?src=\ref[src];close=1'>Close</a>"
+
+	dat += {"<center><h4>[enemy_name]</h4></center>
+		<br><center><h3>[temp]</h3></center>
+		<br><center>Health: [player_hp] | Magic: [player_mp] | Enemy Health: [enemy_hp]</center>"}
+	if (gameover)
+		dat += "<center><b><a href='byond://?src=\ref[src];newgame=1'>New Game</a>"
+	else
+
+		dat += {"<center><b><a href='byond://?src=\ref[src];attack=1'>Attack</a> |
+			<a href='byond://?src=\ref[src];heal=1'>Heal</a> |
+			<a href='byond://?src=\ref[src];charge=1'>Recharge Power</a>"}
+
+	dat += "</b></center>"
+
+	return dat
+
+/datum/arcade_game/space_villain/Topic(href, href_list)
+	if(..())
+		return
+	if (!blocked && !gameover)
+		if (href_list["attack"])
+			action_attack()
+
+		else if (href_list["heal"])
+			action_heal()
+
+		else if (href_list["charge"])
+			action_charge()
+
+	if (href_list["close"])
+		usr.unset_machine()
+		usr << browse(null, "window=arcade")
+
+	else if (href_list["newgame"]) //Reset everything
+		if(is_cheater(usr))
+			return
+
+		temp = "New Round"
+		player_hp = player_max_hp
+		player_mp = player_max_mp
+		enemy_hp = enemy_max_hp
+		enemy_mp = enemy_max_mp
+		gameover = 0
+		turtle = 0
+
+		if(emagged)
+			New()
+			emagged = 0
+
+	holder.add_fingerprint(usr)
+	holder.updateUsrDialog()
+
+
+/datum/arcade_game/space_villain/proc/arcade_action()
+	if ((enemy_mp <= 0) || (enemy_hp <= 0))
+		if(!gameover)
+			gameover = 1
+			temp = "[enemy_name] has fallen! Rejoice!"
+
+			if(emagged)
+				feedback_inc("arcade_win_emagged")
+				new /obj/item/clothing/head/collectable/petehat(holder.loc)
+				new /obj/item/device/maracas/cubanpete(holder.loc)
+				new /obj/item/device/maracas/cubanpete(holder.loc)
+				message_admins("[key_name_admin(usr)] has outbombed Cuban Pete and been awarded explosive maracas.")
+				log_game("[key_name_admin(usr)] has outbombed Cuban Pete and been awarded explosive maracas.")
+				holder.New()
+				emagged = 0
+
+			else if(!holder.contents.len)
+				feedback_inc("arcade_win_normal")
+				var/prizeselect = pickweight(prizes)
+				new prizeselect(holder.loc)
+
+				if(istype(prizeselect, /obj/item/toy/gun)) //Ammo comes with the gun
```

Comment:
Alternatively you could make prizes a list of lists and spawn both the gun and the ammo like that.
Ditto below.

Reply:
done. Turns out the way they've handled that previously was having the syndicate fakespacesuit and fakespacehelmet spawn in a box.